libevmasm/SemanticInformation.cpp

```diff
@@ -430,6 +456,27 @@ SemanticInformation::Effect SemanticInformation::storage(Instruction _instructio
 	}
 }
 
+SemanticInformation::Effect SemanticInformation::transientStorage(Instruction _instruction)
+{
+	switch (_instruction)
+	{
+	case Instruction::CALL:
+	case Instruction::CALLCODE:
+	case Instruction::DELEGATECALL:
+	case Instruction::CREATE:
+	case Instruction::CREATE2:
+	case Instruction::TSTORE:
```

Comment:
Have we decided not to add this to the `UnusedStoreEliminator`, i.e. to not optimize away `TSTORE` and `TLOAD` where possible?

Reply:
I think that will be a future task and in this PR we just want to introduce basic usage/infra of transient storage.

Reply:
Yeah, any optimizations other than basic stuff that will work out of the box (like `UnusedPruner`) are out of scope of this PR.